src/commands/plugins/source.ts

```diff
@@ -29,9 +29,8 @@ export default class PluginsSource extends Command {
   plugins = new Plugins(this.config);
 
   async run() {
-    const remoteCommands: string[] = FeatureFlag.getSingleton()
-      .getFeatureFlagInfo<string[]>("REMOTE_COMMANDS")
-      .sort();
+    const remoteCommands: Array<{ name: string, description: string }> = FeatureFlag.getSingleton()
+    .getFeatureFlagInfo<Array<{ name: string, description: string }>>("REMOTE_COMMANDS_DESCRIPTIONS");
```

Comment:
```suggestion
    .getFeatureFlagInfo<Array<{ name: string, description: string }>>("REMOTE_COMMANDS_DESCRIPTIONS");
```
I would change it to `REMOTE_COMMANDS` instead.

Reply:
👌 nice, let's change it when everything else is ready then. No trouble having a few minutes of the code not working.